src/test/suite/targeting.test.ts

```diff
@@ -136,12 +142,13 @@ suite('targeting.ts', () => {
         });
 
         test('Without selection or active file, throws error', async () => {
-            // ===== SETUP =====
-            // Make sure there's no file open in the editor.
-            const tabs: vscode.Tab[] = vscode.window.tabGroups.all.map(tg => tg.tabs).flat();
-            for (let i = 0; i < tabs.length; i++) {
-                await vscode.window.tabGroups.close(tabs[i]);
-            }
+			// ===== SETUP =====
+			// Simulate no window being open in the editor.
+			// NOTE: We need to use a stub here instead of/in addition to directly closing
+			//       windows, because sometimes the test context can have a weird "phantom window"
+			//       even if you close anything. This seems to happen when the test instance loses
+			//       focus.
+			const windowStub = Sinon.stub(vscode.window, 'activeTextEditor').value(undefined);
```

Comment:
I know this stub seems redundant with line 26. The thing is, there's an odd behavior where if you're running the tests locally and you click away from the window, then the `closeAllEditors` command causes a weird phantom editor to open that doesn't open if the window is still in focus. I could find no information about this phantom editor online.
So in order to allow you to tab away while tests are running, I'm just stubbing out the property as `undefined` to work around that.